warehouse/models/mart/gtfs/dim_attributions.sql

```diff
@@ -32,15 +22,16 @@ dim_attributions AS (
         attribution_email,
         attribution_phone,
         make_dim.base64_url,
-        COALESCE(warning_duplicate_primary_key, FALSE) AS warning_duplicate_primary_key,
+        COUNT(
```

Comment:
Why did you refactor this one away from `bad_rows` but not others

Reply:
Only this one had the null condition in the join and it was causing bad fanout